public/static/locales/fr/form-builder.json

```diff
@@ -363,5 +363,12 @@
   "dyanamicRow": "[FR] Repeating question set",
   "addToSet": "[FR] Add to set",
   "removeFromSet": "[FR] Remove from set",
-  "untitledForm": "Formulaire sans titre"
+  "untitledForm": "Ficher de formulaire sans nom",
+  "questionSet": {
+    "addAnother": {
+      "title": "Bouton",
+      "placeholder": "[FR] Add another",
+      "description": "[FR] Create a descriptive button label to let form fillers know that they can add another set of answers. For example, Add “another address”."
```

Comment:
```suggestion
      "placeholder": "Ajouter un.e autre",
      "description": "Créez une étiquette descriptive pour le bouton afin d'indiquer aux personnes remplissant le formulaire qu'elles peuvent ajouter un autre ensemble de réponses. Par exemple, ajouter « une autre adresses »."
```

Reply:
This might not actually work in French because of gendered language which will need the right article to precede the variable?

For example: 

Add another address = Ajouter **une** autre adresse
Add another email = Ajouter **un** autre courriel

Reply:
Okay - we'll take you commit and go from there - this feature is in development mode for now so it will go to staging first so maybe we can get better feedback on how to handle at that point?